ibm/mas_devops/roles/mongodb/tasks/providers/ibm/backup.yml

```diff
@@ -10,9 +10,9 @@
 - name: "Debug information when backup MongoDB"
   debug:
     msg:
-    - "IBMCloud Region ............. {{ ibmcloud_region }}"
-    - "IBMCloud Resource Group ...... {{ ibmcloud_resourcegroup }}"
-    - "MongoDB Service Name ........ {{ ibm_mongo_name }}"
+      - "IBMCloud Region ............. {{ ibmcloud_region }}"
```

Comment:
Remember: https://github.com/ibm-mas/ansible-devops/blob/master/CONTRIBUTING.md#align-debug-messages

Reply:
Only thing I touched on this file was to fix the lint error, it was merged into master with this misalignment already, but in any case I'm fixing it.